src/components/ButtonGroup/ButtonGroup.tsx

```diff
@@ -50,9 +50,9 @@ export const ButtonGroup = chakra(
       React.Children.map(
         children as JSX.Element,
         (child: React.ReactElement, key: number) => {
-          if (child.type !== Button) {
+          if (child?.type !== Button) {
```

Comment:
Why are these question marks needed here?

Reply:
Just in case it's null/undefined.